src/components/common/ColonyActionsTable/partials/ActionsTableFilters/ActionsTableFilters.tsx

```diff
@@ -59,6 +60,19 @@ const ActionsTableFilters: FC = () => {
     />
   );
 
+  const searchPill = (
+    <div className="flex items-center justify-end rounded-lg bg-blue-100 px-3 py-2 text-sm text-blue-400">
+      <p className="max-w-[12.5rem] truncate sm:max-w-full">{searchFilter}</p>
+      <button
+        type="button"
+        onClick={() => setSearchFilter('')}
+        className="ml-2 flex-shrink-0"
+      >
+        <X size={12} className="text-inherit" />
+      </button>
+    </div>
+  );
```

Comment:
Now also for this pill, it seems to be duplicated across these pages, so could we please extract a new component `SearchPill` - maybe it can be placed in `src/components/v5/common/Pills`

```
interface SearchPillProps {
  value?: string;
  onClick: () => void
}

const SearchPill: FC<SearchPillProps> = ({value, onClick}) => {
  if (!value) {
    return null;
  }

  return (
    <div className="flex items-center justify-end rounded-lg bg-blue-100 px-3 py-2 text-sm text-blue-400">
      <p className="max-w-[12.5rem] truncate sm:max-w-full">{value}</p>
      <button
        type="button"
        onClick={onClick}
        className="ml-2 flex-shrink-0"
      >
        <X size={12} className="text-inherit" />
      </button>
    </div>
  );
}
```